ingestion/tests/cli_e2e/test_cli_bigquery.py

```diff
@@ -209,3 +213,21 @@ def get_test_case_definitions(self) -> List[TestCaseDefinition]:
 
     def get_expected_test_case_results(self):
         return [TestCaseResult(testCaseStatus=TestCaseStatus.Success, timestamp=0)]
+
+    @pytest.mark.order(9999)
```

Comment:
suggest: add comment on why this needs to run last if it's not clear (to it isn't).